tasks.py

```diff
@@ -84,11 +95,7 @@ def __init__(self, task_iters, method, custom_task_ratio=None):
             dataset_ids = [task_iter.dataset.id for task_iter in task_iters]
             task_ratio = [math.sqrt(task_iter.dataset.tensors[0].shape[0])/dataset_ids.count(task_iter.dataset.id) for task_iter in task_iters]
         else:
-            if custom_task_ratio == 'equal':
```

Comment:
Was this deleted on purpose? Or is this just to a mismatch between your branch and master? I believe this was a recent feature

Reply:
I think this was by accident. I'll change it back